aoc2021/build.sbt

```diff
@@ -6,3 +6,5 @@ lazy val root = (project in file("."))
   .settings(
     name := "aoc2021"
   )
+
+libraryDependencies += "com.lihaoyi" %% "pprint" % "0.8.1"
```

Comment:
oh wow - you found the little gem libraries now didn't you ?

Reply:
Nope, Thijs found 'em and recommended them to me. 💯